### PR TITLE
ref bug #133129 Update profile

### PR DESCRIPTION
**Pull request by NguyenTienVinh**
Update user profile when change Group, Branch
display message when click notification
https://dev.framgia.com/redmine/issues/133129

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -82,6 +82,8 @@
     <string name="unpaid_day_off">Unpaid day off</string>
     <string name="number_day_off">Number of day off</string>
     <string name="off_no_salary">Off no salary</string>
+    <string name="this_feature_is_currently_only_available_on_the_web">This feature is currently only available on the website!</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`this_feature_is_currently_only_available_on_the_web`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -82,6 +82,8 @@
...
+    <string name="you_are_unauthorized_to_access">You are unauthorized to access!</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`you_are_unauthorized_to_access`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values-vi/strings.xml [hunk 3]
```diff
@@ -54,6 +54,8 @@
     <string name="confirm_password_does_not_match">Xác nhận mật khẩu không khớp!</string>
     <string name="request_leave">Yêu cầu đi trễ, về sớm</string>
     <string name="can_not_edit_group">Không thể thay đổi phòng ban!</string>
+    <string name="this_feature_is_currently_only_available_on_the_web">Tính năng hiện chỉ khả dụng trên web!</string>
```
Comment (daolq3012) [inline app/src/main/res/values-vi/strings.xml]: Pls change to ``Tính năng xem sở đồ tổ chức hiện chỉ khả dụng ở phiên bản web!``

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -82,6 +82,8 @@
     <string name="unpaid_day_off">Unpaid day off</string>
     <string name="number_day_off">Number of day off</string>
     <string name="off_no_salary">Off no salary</string>
+    <string name="workspace_feature_is_currently_only_available_on_the_web">Workspace\'s information feature is currently only available on the website!</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`workspace_feature_is_currently_only_available_on_the_web`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -82,6 +82,8 @@
...
+    <string name="you_are_unauthorized_to_access">You are unauthorized to access!</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`you_are_unauthorized_to_access`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml (hunk 4 above)
Comment (daolq3012) [inline app/src/main/res/values/strings.xml]: pls change ``Workspace\'s information`` to ``Organization charts feature``

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -82,6 +82,8 @@
     <string name="unpaid_day_off">Unpaid day off</string>
     <string name="number_day_off">Number of day off</string>
     <string name="off_no_salary">Off no salary</string>
+    <string name="workspace_feature_is_currently_only_available_on_the_web">Organization charts feature is currently only available on the website!</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`workspace_feature_is_currently_only_available_on_the_web`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -82,6 +82,8 @@
...
+    <string name="you_are_unauthorized_to_access">You are unauthorized to access!</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`you_are_unauthorized_to_access`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml (hunk 6 above)
Comment (daolq3012) [inline app/src/main/res/values/strings.xml]: pls change key same á values ``workspace_feature_is_currently_only_available_on_the_web`` -> ``organization_charts_feature_is_currently_only_available_on_the_web``

File: app/src/main/java/com/framgia/wsm/screen/main/MainViewModel.java [hunk 8]
```diff
@@ -432,6 +436,15 @@ private void getPageNotification(String notificationRequestType) {
             return;
         }
         switch (notificationRequestType) {
+            case Constant.SILENT_NOTIFICATION:
+                if (mUser.isManage()) {
+                    mDialogManager.dialogError(mContext.getString(
+                            R.string.organization_charts_feature_is_currently_only_available_on_the_web));
```
Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/screen/main/MainViewModel.java]: ERROR: Line is longer than 100 characters (found 106).

File: app/src/main/res/values/strings.xml [hunk 9]
```diff
@@ -82,6 +82,8 @@
     <string name="unpaid_day_off">Unpaid day off</string>
     <string name="number_day_off">Number of day off</string>
     <string name="off_no_salary">Off no salary</string>
+    <string name="organization_charts_feature_is_currently_only_available_on_the_web">Organization charts feature is currently only available on the website!</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`organization_charts_feature_is_currently_only_available_on_the_web`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 10]
```diff
@@ -82,6 +82,8 @@
...
+    <string name="you_are_unauthorized_to_access">You are unauthorized to access!</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`you_are_unauthorized_to_access`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

**Pull request merged**